iOS/SampleApp/supported-parts.txt

```diff
@@ -0,0 +1,27 @@
+"bumper-front",
```

Comment:
damn, why it is in SampleApp, not Framework?

Reply:
in both of them, ok

Reply:
Added as reference in build phases